_posts/2021-05-01-monthly-dev-post.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Monthly Dev Post of May 2021
+author: People
+---
+
+Welcome to your Monthly Dev Post of May 2021.
+Every "month" one of these post will be created, to tell you about the latest developments in the world of OpenTTD.
+
+So I haven't written much lately; so what, neither has Shakespeare.
+
+<!-- more -->
+
+## Development
+
+### Highlights
+
+...
+
+### OpenGL issues / crashes on startup
+
+Although we are really happy with the addition of our OpenGL driver, it also seems not all GPUs are as happy with it as we are.
+We are tackling the issues as they come in and we can reproduce them .. but especially reproducing is a bit of an issue.
+With 1.11.1 we were hoping to fixed most issues, but it seems another subset of (mostly Intel) GPUs seems to say boom now.
+Hopefully soon we have it all figured out.
+
+### Steam release
+
+As mentioned in our [OpenTTD Developers Livestream - #3 - Q&A](https://www.youtube.com/watch?v=kFtUGxIGLlg), the Steam release was a huge success.
+More than we expected, if we are honest.
+The platform seems mostly stable, ignoring the OpenGL issues mentioned above.
+As so many users are using the platform, it also gives us incentive to start looking into further integration with Steam.
+We will get to this back in a bit.
+
+### Networking
+
+Mostly what stood out from the Steam release, how difficult it is to setup a network game.
+And this might sound silly, but it is not something we really realised before.
+Back in 2007, from which this network implementation dates, it was one of the only ways available.
+But as time went on, better and easier ways to join each other in a game have been invented; and we have not caught up with that.
+
+So, a few days after release, we started work on improving the whole network experience.
+This is not an easy task, as there are many moving parts.
+
+The main thing we are addressing, is how multiplayer and singleplayer are more interconnected.
+Don't worry, you still won't need an Internet connection to play OpenTTD, but we are making it more fluent.
+What this mean is, that when you start a game, you can indicate if you either want a private game (offline, nobody can join), friends-only game (your server is not listed but friends can join, see below) or a public game.
+
+- Private games are what singleplayer is now.
+- Friends-only introduces the concept of "invite codes".
+This is a short code you can give to your friends.
+They can enter it in OpenTTD to join your server.
+No fuzz, nothing.
```

Comment:
s/fuzz/fuss/ (I assume)

Reply:
https://www.urbandictionary.com/define.php?term=No%20Fuzz

But yeah, I guess I could use the non-slang variant .. not sure it really matters :D